src/geant_argus/geant_argus/metadata/schema.py

```diff
@@ -0,0 +1,83 @@
+METADATA_V0A2_SCHEMA = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "version": {"const": "v0a2"},
+        "phase": {"type": "string"},
+        "services": {"type": "object"},
+        "severity": {"type": "string"},
+        "endpoints": {
+            "type": "object",
+            "properties": {
+                "juniper": {"type": "array", "items": {"type": "object"}},
+                "coriant": {"type": "array", "items": {"type": "object"}},
+                "infinera": {"type": "array", "items": {"type": "object"}},
+            },
+        },
+        "description": {"type": "string"},
+        "coalesce-count": {"type": "integer"},
+        "endpoint-count": {"type": "integer "},
+    },
+    "required": [
+        "phase",
+        "version",
+        "services",
+        "severity",
+        "endpoints",
+        "description",
+        "coalesce-count",
+        "endpoint-count",
+    ],
+}
+
+METADATA_V0A3_SCHEMA = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "definitions": {
+        "alarm-type": {
```

Comment:
clearer would be to call this "endpoint-type" ... the word "alarm" is already used incorrectly in too many places (e.g. the db table names)